open-api/swagger.yaml

```diff
@@ -1239,6 +1239,10 @@ paths:
           in: query
           type: integer
           description: 'The max node depth returned'
+        - name: constraints
+          in: query
+          type: boolean
+          description: 'Flag if the constraints should be included in the result (always true for hal, defaults to false for json)'
```

Comment:
The difference has to do with the different nature of HAL and JSON: JSON has all children embedded (of course, this can be limited by the depth parameter). If constraints are always included, that blows up the JSON object.
For HAL it is different, it does not include children, but links to children. The constraint objects are only provided for nodes at the top level.